### PR TITLE
Kruglov Evgeniy

### PR DESCRIPTION
**Pull request by circloph**

### CODE REVIEW

File: ops-tools/docker/images/app/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM maven:3.8.4-jdk-11-slim AS build
+COPY target/docker-practice-java-0.0.1-SNAPSHOT.jar /home/app/app.jar
```
Comment (tatiana-vakhrameeva) [inline ops-tools/docker/images/app/Dockerfile]: This line assumes that we have jar file locally. But it means that we have to install java locally to build this jar file. It should be enough to have only docker and docker-compose for running application locally. So now if I clone you repo and try to run docker-compose up I will have a error: file does not exist.

**Review by tatiana-vakhrameeva — COMMENTED** [review]
Great job! I left one comment to dockerfile, please take a look and let me know as it will be fixed

File: ops-tools/docker/images/app/Dockerfile (hunk 1 above)
Comment (circloph) [reply ops-tools/docker/images/app/Dockerfile]: Fixed remarks

Comment (tatiana-vakhrameeva) [reply ops-tools/docker/images/app/Dockerfile]: Cool, thank you! Your work is accepted

**Pull request closed without merging**